### DISCUSSION TITLE
Does anyone have examples of how to embed this in svelte or sveltekit?

### DISCUSSION THREAD

**Discussion by ddxv** [mkdocs/mkdocs#3592] [Q&A] [answered]
I have gotten stuck trying to embed this in sveltekit due to the routes overlapping with the exported `docs/` directory. I'd love if someone knows of a working example I could look at.

**Comment by Andre601** [comment]
What exactly do you mean with "embed"?
And why do you actually want that?

Also, you can change the source directory for MkDocs with the `docs_dir` setting in the mkdocs.yml file.

**Reply by ddxv** [reply]
@Andre601 By embed I mean that the mkdocs docs html files would be within a larger site managed by Svelte, for example a menu bar at the top managed by sveltekit. 

I tried the `docs_dir` but thought that was for having a different directory for the `.md` files? I also tried `site_dir` for a custom export location for the `.html` and while I could get the main `index.html` to load the `css` and other files seemed to be loading incorrectly.

**Reply by Andre601** [reply]
Yes, the `docs_dir` is for the MD files and `site_dir` would be for the file to export the HTML files to.

Not sure about the CSS. If I recall correctly are usually relative links generated (Using the `url` filter in jinja2 template files), so if you move the generated files arount are chances high you may have broken some relative links here...

**Reply by ddxv** [reply]
@Andre601 Thanks for your help, it's kinda a dance between Svelte and mkdocs, so struggling a bit to make it work.

For reference, here is how far I've gotten currently:
https://openattribution.dev/

Going to the site, then clicking "Documentation" works, but clicking on the links inside there end up directly loading the Sveltekit `/static/` directory and I lose the menu bar.

When I try to change the export directory of mkdocs to something like `/static/mydocs/` instead and try embedding html in `/docs/` iframe with `srcdoc=` the text from `/static/mydocs/index.html` I get lots of errors for the secondary files that try to load.

**Comment by thomas-mc-work** [comment, marked as answer]
In case you're still looking for an approach: I've created a minimal example project consiting of a mkdocs project with a Svelte integration (without Kit):

https://codeberg.org/common-good/mkdocs-svelte-integration
